deploy/crds/siddhi_v1alpha1_siddhiprocess_cr.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: siddhi.io/v1alpha1
```

Comment:
Shall we rename the CRD instantiation saying this is an example. Also update the content as well. Let's have the name as example-siddhi-deployment